pmm-app/src/pmm-qan/panel/components/Details/Explain/Explain.tools.ts

```diff
@@ -28,7 +28,7 @@ export const processClassicExplain = (classic): ClassicExplainInterface => {
     .map((e) => (String(e) ? e.trim() : ''))
     .filter(Boolean)
     .reduce((acc, row, index) => {
-      acc[headerList[index].accessor] = row;
+      acc[headerList[index].accessor] = row.replaceAll(',', ', ');
```

Comment:
this kind of workaround should be explained. I don't think it is a proper fix. Why do you just use css?

Reply:
The initial/default value of a CSS property 'white-space' is 'normal', so we do not need to change it.
With this value inside the elements the lines are broken by space as necessary to fill the element (the cell in this case).
Resolving the second part of the ticket,- adding spaces after commas,- works for the text overflow as well.

Reply:
I propose not to modify that.

Reply:
Fixed.